docs(cloud-automation): Updated compose-services and cloud-automation (#67)
uc-cdis/gen3.org: content/resources/operator/_index.md

# Running a Gen3 Data Commons

## 1. Compose Services ![Compose Services](img/compose-services.svg)

Compose Services allow you to quickly run a Gen3 commons on a single computer or in a single VM in a matter of minutes.

This option is suitable for those that wish to run a small data commons, experiment with the technology, or even do local development on the Gen3 services.

> We recommend you start here for your first experience running the Gen3 platform. Using Compose Services does not get you the full suite of automation, and logging for Gen3 commons. If you are considering to create bigger data commons or deploy data commons in production environments, we strongly recommend you to use [Cloud Automation](#2-cloud-automation) for these purposes.

Reviewer: ```suggestion
We recommend you start here for your first experience running the Gen3 platform. Using Compose Services will not make the full suite of automation and logging for Gen3 commons available. If you are considering to create bigger data commons or deploy data commons in production environments, we strongly recommend you use [Cloud Automation](#2-cloud-automation) for these purposes.
```

Author: Thanks! Changed in commit 97d1885
